frontend/src/app/features/calendar/sidemenu/calendar-sidemenu.component.ts

```diff
@@ -25,10 +26,12 @@ export class CalendarSidemenuComponent extends UntilDestroyedMixin {
 
   @Input() projectId:string|undefined;
 
-  canCreateCalendar$ = this.currentUserService.hasCapabilities$(
-    'calendars/create',
-    this.currentProjectService.id || undefined,
-  )
+  canCreateCalendar$ = this
+    .capabilitiesService
+    .hasCapabilities$(
```

Comment:
It's a bit weird that a general `capabilitiesService.hasCapabilities$` returns capabilities for the current user only. Is there some way the capabilities for another user or group might end up in that service? In that case I'd slightly change the naming.

Reply:
Right now, this is not possible as the the current user is always passed. It would be possible to call `fetchCapabilities` with a different user filter, but I'm not aware we're using that right now.

I'll create a separate PR for using this service in the context of team planner, were caps are loaded manually right now